Sources/PT.PM.Common/Nodes/Tokens/BinaryOperator.cs

```diff
@@ -41,6 +41,10 @@ public enum BinaryOperator
 
         // JavaScript
         In,               // in
-        InstanceOf        // instanceof
+        InstanceOf,       // instanceof
+
+        //Python
```

Comment:
Additionally Python 3 now has `@` binary operator, shouldn't we handle it as well? Or will it interfere with decorators (attributes)?

Reply:
`@` in Python 3 is supposed to represent matrix multiplication. But as long as operators are overridable in user classes it can stand for anything.
Decorators in Python start with `@`, but the context is completely different for sure.

Reply:
yeah, it's numpy operator for numpy arrays. We should handle this